dist/neo.blockchain.node.js

```diff
@@ -460,100 +396,65 @@ module.exports = function(network) {
     *  }
     * @returns {Promise.<Object>} An object containing the transaction information.
     */
-    this.getRawTransaction = function(txid){
-
-      return new Promise(function(resolve, reject){
-        node.call({
-          method: "getrawtransaction",
-          params: [txid,1],
-          id: 0
-        })
-        .then(function (data) {
-          resolve(data.result);
-        })
-        .catch(function (err) {
-          reject(err);
-        })
-      })
-    };
+    this.getRawTransaction = (txid) => new Promise(function (resolve, reject) {
+      node.call({
+        method: 'getrawtransaction',
+        params: [txid,1],
+        id: 0
+      }).then(({ result }) => {
+        resolve(result)
+      }).catch(reject())
```

Comment:
Parenthesis should be removed. The intent was to pass the function reject, not the value